skrub/_table_vectorizer.py

```diff
@@ -536,6 +542,9 @@ def add_step(steps, transformer, cols, allow_reject=False):
         cols = s.all() - self._specific_columns
 
         self._preprocessors = [CheckInputDataFrame()]
+        if self.drop_null_columns:
+            add_step(self._preprocessors, DropNullColumn(), cols, allow_reject=True)
```

Comment:
- we may want to insert it after CleanNullStrings? so that if the column becomes full of nulls after converting `"N/A"` to `null` it will be dropped. also it's not important but your transformer never raises a RejectColumn exception so `allow_reject` has no effect you don't need it here and can leave the default

Reply:
I added it after CleanNullStrings, but I think I did it in an ugly way, maybe it can be fixed